modules/core/src/parallel.cpp

```diff
@@ -618,9 +618,9 @@ int getNumThreads(void)
 
 #elif defined HAVE_CONCURRENCY
 
-    return 1 + (pplScheduler == 0
+    return (pplScheduler == 0)
         ? Concurrency::CurrentScheduler::Get()->GetNumberOfVirtualProcessors()
-        : pplScheduler->GetNumberOfVirtualProcessors());
+        : pplScheduler->GetNumberOfVirtualProcessors();
```

Comment:
```
    return (pplScheduler == 0)
        ? Concurrency::CurrentScheduler::Get()->GetNumberOfVirtualProcessors()
        : (1 + pplScheduler->GetNumberOfVirtualProcessors());
```

(you can check that through `--test_threads=2` command-line parameter for OpenCV tests)